css/main.scss

```diff
@@ -2538,3 +2538,43 @@ color: #FFA300;
 	margin-top: 40px!important;
 }
 
+#fundraising_heading {
```

Comment:
maybe it shouldn't be a part of `main.scss`? i would create `fundraising.scss` or put this code into `others.scss`. what do you think?

Reply:
Yeah, that's ok. Will do like that.